### PR TITLE
Tidy analyzers

### PR DESCRIPTION
**Pull request by cliffckerr**
### Brief description
Removes the `timeseries_recorder` and `verbose_sim` analyzers, which I don't think anyone is using, and which keep causing tests to fail. (If we want this functionality, I think it would be better to re-implement it.) This PR also refactors the calibration test, so it shouldn't fail randomly now (before it was basically just fitting noise).

### Type(s) of change
- [ ] Bugfix
- [x] Refactor
- [ ] New feature
- [ ] Other

### Checklist
- My code follows the [style guide](https://github.com/amath-idm/styleguide)
  - [x] Yes
  - [ ] N/A
- I've commented my code
  - [x] Yes
  - [ ] N/A
- I've incremented the version number
  - [x] Yes
  - [ ] N/A
- I've updated the changelog
  - [x] Yes
  - [ ] N/A
- I've added tests and checked code coverage
  - [x] Yes
  - [ ] N/A

### CODE REVIEW

File: fpsim/version.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = '0.26.7'
-__versiondate__ = '2023-10-24'
+__version__ = '0.26.8'
```
Comment (robynstuart) [inline fpsim/version.py:1]: what would it take to get to 1.0? 😮

**Review by robynstuart — APPROVED** [review]
Looks good!

File: fpsim/version.py (hunk 1 above)
Comment (cliffckerr) [reply fpsim/version.py:1]: That's definitely a @MObrien-IDM question :grin:

**Review by MObrien-IDM — APPROVED** [review]
Thanks for doing this, Cliff! LGTM

**Pull request merged**